src/test/java/com/varchenko/InsertSortTest.java

```diff
@@ -0,0 +1,23 @@
+package com.varchenko;
+
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+class InsertSortTest {
+    private final InsertSort insertSort = new InsertSort();
```

Comment:
Please add tests for `null` and for passing empty array

Reply:
done